src/content/en/2024/third-parties.md

```diff
@@ -3,17 +3,205 @@
 title: Third Parties
 description: Third Parties chapter of the 2024 Web Almanac covering data of what third parties are used on the web and a deep dive into preventing them causing negative performance impacts.
 hero_alt: Hero image of Web Almanac characters plugging various things into a web page.
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [turban1988, Yash-Vekaria, zubairshafiq, ChrisBeeti]
+reviewers: [tunetheweb]
+editors: [neriiavr]
+analysts: [Yash-Vekaria, ChrisBeeti]
 translators: []
+turban1988_bio: TODO
+Yash-Vekaria_bio: TODO
+zubairshafiq_bio: TODO
+ChrisBeeti_bio: TODO
 results: https://docs.google.com/spreadsheets/d/18uTDBygSqgT_PNFldOz4guLSuXyMzDthRGnAG5if4sU/
-featured_quote:
-featured_stat_1:
-featured_stat_label_1:
-featured_stat_2:
-featured_stat_label_2:
-featured_stat_3:
-featured_stat_label_3:
+featured_quote: Google is the most popular third-parties on the web, with five of the top ten third-party domains being Google domains.
+featured_stat_1: 92%
+featured_stat_label_1: Web pages including one or more third parties
+featured_stat_2: 78
+featured_stat_label_2: The median number of third-party requests per web page
+featured_stat_3: 22
+featured_stat_label_3: The median number of third-parties per web page
+doi: TODO
```

Comment:
@nrllh can you please provide the doi for this chapter?

Reply:
Sure, here: `10.5281/zenodo.14193383`

Reply:
@tunetheweb just updated the DOI